src/phy/fault_injector.rs

```diff
@@ -188,12 +188,12 @@ impl<D: Device> Device for FaultInjector<D>
     type RxBuffer = D::RxBuffer;
     type TxBuffer = TxBuffer<D::TxBuffer>;
 
-    fn limits(&self) -> DeviceLimits {
```

Comment:
Do you think you could finish the renaming in this commit? I'd like to keep history bisectable, and generally clean.